src/pages/popup/components/EditCourseDialog.tsx

```
@@ -0,0 +1,71 @@
+
+interface EditCourseDialogProps {
+  open?: Course;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  onConfirm: (course?: Course) => void;
+}
+
+const EditCourseDialog = (props: EditCourseDialogProps): ReactNode => {
+  const [courseName, setCourseName] = useState(props.open?.name);
+
+  const handleSubmit = () => {
+    if (courseName && courseName.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)